src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonService.kt

```diff
@@ -78,16 +79,16 @@ class PrisonService(
       )
       .offenderNo
 
-  fun transferInFromCourt(confirmArrivalDetail: ConfirmArrivalDetail, arrival: Arrival): Long {
+  fun returnFromCourt(confirmCourtReturnRequest: ConfirmCourtReturnRequest, arrival: Arrival): Long {
 
     return prisonApiClient.courtTransferIn(
       arrival.prisonNumber!!,
-      with(confirmArrivalDetail) {
+      with(confirmCourtReturnRequest) {
         CourtTransferIn(
           prisonId!!,
-          movementReasonCode,
-          commentText,
-          bookingInTime
+          null,
```

Comment:
default these to null in the `CourtTransferIn` so don't have to pass them in here?

Reply:
done